openpype/settings/defaults/project_settings/maya.json

```
@@ -500,7 +500,24 @@
                     "shadows": true,
                     "textures": true,
                     "twoSidedLighting": true,
+                    "lineAAEnable": true,
                     "ssaoEnable": true,
+                    "ssaoAmount": 1,
+                    "ssaoRadius": 16,
+                    "ssaoFilterRadius": 16,
+                    "ssaoSamples": 16,
+                    "fogging": true,
```

I think this should be off by default
```suggestion
                    "fogging": false,
```